pages/EditProfile/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
I don't think we will need an EditProfile page since we want in-line editing. Please delete this file.